source/fundamentals/connection/tls.txt

```diff
@@ -66,6 +66,43 @@ using a method in the ``MongoClientSettings.Builder`` class.
                 .build();
          MongoClient client = MongoClients.create(settings);
 
+Configure Netty
+~~~~~~~~~~~~~~~
+
+If you use the driver with `Netty <https://netty.io/>`__ for network IO,
+you have an option to plug an alternative TLS/SSL protocol implementation
+provided by Netty.
+
+.. code-block:: java
+   :copyable: True
```

Comment:
```suggestion
   :copyable: true
```
It looks like other `copyable` usages [in this repo](https://github.com/mongodb/docs-java/search?q=copyable) use a lowercase first letter. Have you been writing Python recently?